dbt/models/qc/schema.yml

```diff
@@ -310,319 +318,319 @@ models:
               meta:
                 table_name: asmt
-                description: val34 should not be negative
+                description: val34 (BOR FACT MKT PREF LD) should not be negative
               <<: *test-non-negative
       - name: val35
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val35_gte_0
               meta:
                 table_name: asmt
-                description: val35 should not be negative
+                description: val35 (ASSD FARMLAND) should not be negative
               <<: *test-non-negative
       - name: val36
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val36_gte_0
               meta:
                 table_name: asmt
-                description: val36 should not be negative
+                description: val36 (.ASSD FARM BUILDING) should not be negative
               <<: *test-non-negative
       - name: val37
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val37_gte_0
               meta:
                 table_name: asmt
-                description: val37 should not be negative
+                description: val37 (STATE ASSD POLL CONT) should not be negative
               <<: *test-non-negative
       - name: val38
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val38_gte_0
               meta:
                 table_name: asmt
-                description: val38 should not be negative
+                description: val38 (STATE ASSD RAILROADS) should not be negative
               <<: *test-non-negative
       - name: val39
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val39_gte_0
               meta:
                 table_name: asmt
-                description: val39 should not be negative
+                description: val39 (BOR FACT LAND) should not be negative
               <<: *test-non-negative
       - name: val40
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val40_gte_0
               meta:
                 table_name: asmt
-                description: val40 should not be negative
+                description: val40 (TOTAL ASSESSED VALUE) should not be negative
               <<: *test-non-negative
       - name: val41
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val41_gte_0
               meta:
                 table_name: asmt
-                description: val41 should not be negative
+                description: val41 (BOR FACT BLDG) should not be negative
               <<: *test-non-negative
       - name: val42
         tests:
           - accepted_range:
               name: qc_vw_neg_asmt_value_val42_gte_0
               meta:
                 table_name: asmt
-                description: val42 should not be negative
+                description: val42 (40. .:) should not be negative
```

Comment:
Some of these fields don't have verbose names in iasWorld; in this case, `val42` maps to a single dot (`.`). I've styled it as `40. .:` to more closely match the way the field appears in the UI, but I'm open to alternatives if this is confusing.

I wonder if the lack of a name is an indication that this field isn't used?

Reply:
I wish I knew. I assume it's unused but we should confirm with MV